lib/ansible/modules/cloud/azure/azure_rm_image_facts.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_image_facts
+
+version_added: "2.7"
+
+short_description: Get facts about azure custom images.
+
+description:
+    - List azure custom images. The images can be listed where scope of listing can be based on subscription, resource group, name or tags.
+
+options:
+    resource_group:
+        description:
+            - Name of resource group.
+    name:
+        description:
+            - Name of the image to filter from existing images.
+    tags:
+        description:
+            - List of tags to be matched.
+    format:
```

Comment:
For the new added facts, we suggests no `raw`

Reply:
Please correct me if I am wrong, so does this mean we should remove support for ```raw``` and keep only curated format? (something similar like https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/azure/azure_rm_postgresqldatabase_facts.py ?)

Reply:
Yes, it is. Some modules contain both raw and curated because of the compatible issue